assets/scss/themes/default.scss

```diff
@@ -42,9 +42,44 @@ $bootstrap-colors: (
   "danger"
 );
 
-@import '_bootstrap';
-
 .admin__sidebar {
   background: rgb(42,57,70);
   background: linear-gradient(-70deg, rgba(47,62,77,1) 50%, rgba(42,57,70,1) 50.5%);
+}*/
+
+$theme-colors: (
+  "primary": #497081,
+  "secondary": #2F3E4D,
+  "tertiary": #5b8394,
+  "quaternary": #7298a8
+);
+
+// Disable Tiramisu
+$enable-rounded: false;
+$enable-shadows: false;
+$enable-gradients: false;
+
+// CSS
+// -------------------
+
+// Toolbar
+.svg-brand {
+  min-width: 20px;
+  min-height: 20px;
+  background-image: url('../../static/brand/bolt4-icon.svg');
```

Comment:
Seems like this one's missing. :-)

Reply:
git add is not my friend.

Reply:
Ping me on slack, if you'd like some pointers! :-)